superbit_lensing/galsim/mock_superBIT_data.py

```diff
@@ -767,6 +829,29 @@ def main():
 
     logprint('Made telescope PSF profile')
 
+    ###
+    ### Make generic WCS
+    ###
+
+    # If you wanted to make a non-trivial WCS system, could set theta to a non-zero number
+    fiducial_full_image = galsim.ImageF(sbparams.image_xsize, sbparams.image_ysize)
+
+    theta = 0.0 * galsim.degrees
+    dudx = np.cos(theta) * sbparams.pixel_scale
+    dudy = -np.sin(theta) * sbparams.pixel_scale
+    dvdx = np.sin(theta) * sbparams.pixel_scale
+    dvdy = np.cos(theta) * sbparams.pixel_scale
+
+    affine = galsim.AffineTransform(dudx, dudy, dvdx, dvdy, origin=fiducial_full_image.true_center)
+    sky_center = galsim.CelestialCoord(ra=sbparams.center_ra, dec=sbparams.center_dec)
+    wcs = galsim.TanWCS(affine, sky_center, units=galsim.arcsec)
+
+    ##
+    ## Define RNG for dither offsets
+    ##
+    rng = np.random.default_rng()
```

Comment:
Maybe add it as a required seed in `_set_seeds()` so this is predictable?

Reply:
Have we checked that this would lead to `nexp` independent dithers, or will there be "clumps" of offsets? Again, I'm not sure that we ever satisfactorily solved the multiprocessing + `numpy.random.rng()` issue. Perhaps this won't be an issue, since the `rng()` is called before a job is passed to the `pool`.  I'll run a unit test in my local `dithering` and update here.

Reply:
Yes I think it isn't an issue here as this is all done outside of the jobs pool. Thanks for checking!